dashboard/app/helpers/district_dropdown_constants.rb

```diff
@@ -0,0 +1,8 @@
+module DistrictDropdownConstants
```

Comment:
In that case it might actually make the most sense as its own model. The user and enrollment can either have a district or not, but a district entry will always be self-consistent by validations in that model. What do you think? 

I don't know if we want to do this, but we could even normalize it such that any given school/district combination is only stored once and then just pointed to by user and enrollment.